### PR TITLE
[Changes Requested] #30 #38 Added

### PR DESCRIPTION
**Pull request by slavahatnuke**
#30 #38

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo ok && exit 0"
+    "test": "echo ok && exit 1"
```
Comment (slavahatnuke) [inline package.json]: hey what is it?

File: test-1.txt [hunk 2]
```diff
@@ -1 +1 @@
-2017-06-27T00:27:24.599Z
+2017-06-28T15:48:05.425Z
```
Comment (slavahatnuke) [inline test-1.txt]: Hm, ok!

File: go.sh [hunk 3]
```diff
@@ -3,5 +3,5 @@ node -e "console.log(new Date())" > test-1.txt
 node -e "console.log(new Date())" > test-2.txt
 
 git add .
-git commit -m "#1 Added"
+git commit -m "#38 Added"
```
Comment (slavahatnuke) [inline go.sh]: ok

File: test-1.txt (hunk 2 above)
Comment (slavahatnuke) [inline test-1.txt]: Hey

File: test.txt [hunk 4]
```diff
@@ -1 +1 @@
-2017-06-27T00:27:24.508Z
+2017-06-28T15:48:05.329Z
```
Comment (alexkorsun) [inline test.txt]: this looks good

File: test-2.txt [hunk 5]
```diff
@@ -1 +1 @@
-2017-06-27T00:27:24.691Z
+2017-06-28T15:48:05.518Z
```
Comment (alexkorsun) [inline test-2.txt]: lgtm

**Pull request merged**